### PR TITLE
Token-container tests are skipped because of pulsar shutdown tests.

### PR DESCRIPTION
**Pull request by vogdb**
There was a problem that the last test `user-container.js` was never executed because the test before it `pulsar.js` had to call `process.exit` in order to test graceful shutdown of API.
The solution I took is to reorganize the tests, so `pulsar.js` will be the last to execute.

### CODE REVIEW

**Comment by vogdb** [comment]
@tomaszdurka please review.

File: test/spec/exec.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-var PulsarExec = require('../lib/pulsar/exec');
+var PulsarExec = require('../../lib/pulsar/exec');
 var assert = require('chai').assert;
-var jobArgs = require('./job-args');
+  var jobArgs = require('../data/job-args');
```
Comment (tomaszdurka) [inline test/spec/exec.js]: Intend

**Comment by vogdb** [comment]
@tomaszdurka please review.

**Comment by tomaszdurka** [comment]
Looks good

**Pull request merged**